src/Conditionals.elm

```diff
@@ -0,0 +1,544 @@
+module Conditionals exposing (..)
+
+
+calculateGrade : Int -> String
+calculateGrade score =
+    if score >= 100 && score <= 0 then
+        "Invalide Score"
+        -- If score is greater than or equal to 100 and score
+        -- is less than or equal 0  "Invalide Score" will be assign as the grade.
+
+    else if score >= 90 then
+        "A"
+        --If score is greater than or equal 90 "A"  will be assign as the grade.
+
+    else if score >= 80 then
+        "B"
+        --If score is greater than or equal 80 "B" will be assign as the grade.
+
+    else if score >= 70 then
+        "C"
+        --If score is greater than or equal 70 "C" will be assign as the grade.
+
+    else if score >= 60 then
+        "D"
+        --If the score greater than or equal 60 "D" will be assign as the grade.
+
+    else
+        "F"
+
+
+
+--Anything else from 59 to 0 should return "F"
+-- This function  takes an Int and returns a string,
+-- Then classify ages into different groups.
+
+
+categorizeCharacter : Char -> String
+categorizeCharacter char =
+    if Char.isAlpha char then
+        let
+            lowercaseChar =
+                Char.toLower char
+
+            --This line handles both uppercase and lowercase characters.
+            --And it also check if the character is alphabetic,
+            --It converts the character to lowercase using Char.toLower.
+        in
+        if List.member lowercaseChar [ 'a', 'e', 'i', 'o', 'u' ] then
+            "Vowel"
+            --This line checks if the listed character member here is alphabetic,
+            -- It categorizes it as a "Vowel"; else any character
+            --Outside of this box, is categorizes as a "Consonant".
+
+        else
+            "Consonant"
+
+    else if Char.isDigit char then
+        "Digit"
+        --If the character is not alphabetic but is a digit, from ('0' to '9')
+        -- It categorizes it as a "Digit
+
+    else
+        "Unknown"
```

Comment:
Hey, this is beyond what we've learnt so far.

If I were to ask you to talk me through how you solved this, and gave you a similar question to solve this way would you be able to?

Just a curious question.

Reply:
Also, in my former reviews I suggested a way for you to tackle this. Can you please look at that and implement it?

Reply:
> Also, in my former reviews I suggested a way for you to tackle this. Can you please look at that and implement it?



> Hey, this is beyond what we've learnt so far.
> 
> If I were to ask you to talk me through how you solved this, and gave you a similar question to solve this way would you be able to?
> 
> Just a curious question.

Yes I can, and I thought you said the first one is too  tedious, that was why I listed just the "Vowels" and made  everything else like other alphabet that is not listed "Consonant"
But I will rewrite the code again the way I wrote the first "Digit"